.github/workflows/style.yaml

```diff
@@ -0,0 +1,33 @@
+name: Style
+on:
+  push:
+    tags:
+    - '*'
+    branches:
+    - master
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+jobs:
+  check-generated-files:
+    env:
+      ARTIFACT_DIR: junit-reports/
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.61
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - uses: ./.github/actions/job-preamble
```

Comment:
Is there a way to get this from [stackrox/stackrox](https://github.com/stackrox/stackrox)? If so, would it make to do that?

Reply:
I imagine we could publish the Action just like any other reusable GH Action.